pkg/csl/reader_uk.go

```diff
@@ -1,31 +1,37 @@
 package csl
 
 import (
+	"bytes"
 	"encoding/csv"
 	"errors"
 	"fmt"
 	"io"
 	"os"
 	"strconv"
 	"strings"
+
+	"github.com/knieriem/odf/ods"
```

Comment:
Q: Is there another package that can read open document files? I'm a bit worried to import this code as-is.

Reply:
The list of libraries that read open document files is pretty slim, but I can take a look at another one.

Reply:
I read the code a bit more and this library is pretty small. It's fine for now, but I am going to try and fuzz it a little.